MergeFRCPipeline.py

```diff
@@ -380,7 +382,9 @@ def switchCam():
     #PipeLine Table Values, Unique for Each PipeLine
     networkTableVisionPipeline.putBoolean("Driver", False)
     networkTableVisionPipeline.putBoolean("Tape", TapeEnabled)
-    networkTableVisionPipeline.putBoolean("PowerCell", PowerCellEnabled)
+    networkTableVisionPipeline.putBoolean("Cargo", CargoEnabled)
+    networkTableVisionPipeline.putBoolean("Red", RedEnabled)
+    networkTableVisionPipeline.putBoolean("RedBlue", BlueEnabled)

```

Comment:
Should this be "Blue" instead of "RedBlue"?

Reply:
You are correct